cli/ts/proveOnChain.ts

```diff
@@ -380,6 +375,7 @@ const proveOnChain = async (args: any) => {
             return 
         }
 
+        const txErr = 'Error: processMessages() failed'
```

Comment:
Another consistency nitpick - looks like you're using a mix of single quotes & double quotes. Do we have a convention on this? I don't have a strong preference... just that we be consistent